source/about/mattermost-server-releases.md

```diff
@@ -17,9 +17,9 @@ Mattermost releases a new server version on the 16th of each month in [binary fo
 
 ## Latest releases
 
-| **Release** | **Released on** | **Support ends** |
-|:---|:---|:---|
-| v10.4 [Download](https://releases.mattermost.com/10.4.1/mattermost-10.4.1-linux-amd64.tar.gz) \| {ref}`Changelog <release-v10.4-feature-release>` | 2025-01-16 | 2025-04-15 |
+| **Release** | **Released on** | **Support ends** | **SBOM Team Edition Download Link** |
+|:---|:---|:---|:---|
+| v10.4 [Download](https://releases.mattermost.com/10.4.1/mattermost-10.4.1-linux-amd64.tar.gz) \| {ref}`Changelog <release-v10.4-feature-release>` | 2025-01-16 | 2025-04-15 | [SBOM download](https://github.com/mattermost/mattermost/releases/download/v10.4.1/sbom-mattermost-v10.4.1.json) |
```

Comment:
That's just another option if we prefer not having an extra column. @amyblais I'll leave this to you and @cwarnermm to decide. 

```suggestion
| **Release** | **Released on** | **Support ends** | 
|:---|:---|:---|
| v10.4 [Download](https://releases.mattermost.com/10.4.1/mattermost-10.4.1-linux-amd64.tar.gz) \| {ref}`Changelog <release-v10.4-feature-release>` \| [SBOM download](https://github.com/mattermost/mattermost/releases/download/v10.4.1/sbom-mattermost-v10.4.1.json) | 2025-01-16 | 2025-04-15 | 
```

Reply:
@amyblais - I like the idea of keeping our existing 3 columns with @esarafianou's suggestion. Any concerns from your perspective?